### PR TITLE
#628: Text-area for string sequence inputs

### PR DESCRIPTION
**Pull request by jozefbakus**
Implements #628

### CODE REVIEW

File: ui/src/app/components/workflows/workflow-form/dynamic-parts/string-sequence-part/string-sequence-part.component.html [hunk 1]
```diff
@@ -14,35 +14,57 @@
...
+                [lengthValidator]="{min: minLength, max: maxLength}"></textarea>
+      <clr-control-error *clrIfError="'nonEmptyValidator'">{{texts.FORM_VALIDATION_MUST_BE_FILLED(name)}}</clr-control-error>
+      <clr-control-error *clrIfError="'lengthValidator'">{{texts.FORM_VALIDATION_MIN_MAX_LENGTH(name, minLength, maxLength)}}</clr-control-error>
+    </clr-textarea-container>
+  </div>
+
+  <clr-toggle-container>
+    <label></label>
+    <clr-checkbox-wrapper>
+      <input [name]="name+uiid+'freetexttoogle'" type="checkbox" clrToggle [(ngModel)]="isFreeText"/>
```
Comment (kevinwallimann) [inline ui/src/app/components/workflows/workflow-form/dynamic-parts/string-sequence-part/string-sequence-part.component.html]: nit: `toogle` -> `toggle`

File: ui/src/app/components/workflows/workflow-form/dynamic-parts/string-sequence-part/string-sequence-part.component.html [hunk 2]
```diff
@@ -14,35 +14,57 @@
...
+      </button>
+    </clr-control-container>
+  </div>
+
+  <div *ngIf="isFreeText">
+    <clr-textarea-container>
+      <label>{{name}}</label>
+      <textarea #freeTextInput [name]="name+uiid+'freetext'" [ngModel]="getFreeTextInputValue()"
+                (ngModelChange)="freeTextInputChange($event)" clrTextarea
+                [nonEmptyValidator]="isRequired"
```
Comment (kevinwallimann) [inline ui/src/app/components/workflows/workflow-form/dynamic-parts/string-sequence-part/string-sequence-part.component.html]: Can't you use `[required]="isRequired"` here as well? Why do you need a custom NonEmptyValidator?

**Comment by kevinwallimann** [comment]
Nice feature, btw. I like it a lot 🚀

File: ui/src/app/components/workflows/workflow-form/dynamic-parts/string-sequence-part/string-sequence-part.component.html (hunk 1 above)
Comment (jozefbakus) [reply ui/src/app/components/workflows/workflow-form/dynamic-parts/string-sequence-part/string-sequence-part.component.html]: Done

File: ui/src/app/components/workflows/workflow-form/dynamic-parts/string-sequence-part/string-sequence-part.component.html (hunk 2 above)
Comment (jozefbakus) [reply ui/src/app/components/workflows/workflow-form/dynamic-parts/string-sequence-part/string-sequence-part.component.html]: Yes, you are right. I reinvented a wheel. Fixed

**Review by kevinwallimann — APPROVED** [review]

**Pull request merged**